### PR TITLE
TimerLog: avoid sstream in header

### PR DESCRIPTION
**Pull request by akva2**
minor minor, but still better.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/common/OpmLog/TimerLog.hpp [hunk 1]
```diff
@@ -50,7 +50,6 @@ class TimerLog : public StreamLog {
                                    const std::string& message) override;
 private:
     clock_t m_start;
-    std::ostringstream m_work;
```
Comment (bska) [inline opm/common/OpmLog/TimerLog.hpp:53]: Cool.  Do you plan to move [the `<sstream>` header](https://github.com/OPM/opm-common/blob/91e2a1256e5cf0ef0dc51e4c9be0ccf37a12e91d/opm/common/OpmLog/TimerLog.hpp#L25) to the implementation file too?

**Review by bska — APPROVED** [review]
I like it.  Please merge if you're satisfied with the state of this and the build check is green.  I see one potential additional step, but I suppose you plan to do that in follow-up work.

File: opm/common/OpmLog/TimerLog.hpp (hunk 1 above)
Comment (akva2) [reply opm/common/OpmLog/TimerLog.hpp:53]: Yeah, that was sort of the whole point. Bad git add -p work on my behalf..

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Very good.  Please merge when the build check is green.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**